src/søknad/forside/RedirectTilStart.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import {
+  Route,
+  Redirect,
+  RouteProps,
+  RouteComponentProps,
+} from 'react-router-dom';
+import { useSøknad } from '../../context/SøknadContext';
+
+interface RedirectTilStartProps extends RouteProps {
+  component:
+    | React.ComponentType<RouteComponentProps<any>>
+    | React.ComponentType<any>;
+}
+const RedirectTilStart: React.FC<RedirectTilStartProps> = ({
+  component: Component,
+  ...rest
+}) => {
+  const { søknad } = useSøknad();
+  return (
+    <Route
+      {...rest}
+      render={(props) =>
+        søknad.harBekreftet === false ? (
```

Comment:
To muligheter her - hva er best?
1) eksplisitt `false`-sjekk og `undefined` som lovlig verdi
2) implisitt `true/false` og `false` som default verdi

Ulempe med nr 2 er at alle testerne sine mellomlagrede søknader må nulles ut (modellversjon må bumpes)...

Reply:
Hmm, er det en grunn til at vi vil ha `undefined` som lovlig verdi? 
Hvis ikke, kunne vi brukt 
```
!søknad.harBekreftet ? (...
```
Da vil både false & undefined føre til en redirect, tror jeg 😄 
På en annen side, skal vi ta høyde for at man ikke kan gå direkte til andre steg via urlen hvis man allerede er på f.eks. steg-2 allerede? 

Det blir sikkert litt mer hassle, men da vil man vel også ha samme case at man har hoppet over et steg og ikke har data derfra. Det er sikkert sjeldent at brukere gjør dette hvis de faktsik ønsker å søke på stønaden, men noe som kan skje 🤷‍♀️

Reply:
Går du direkte til steg X med å endre url så nullstilles tilstanden din - da må du isåfall bruke "back"/"forward" i nettleseren, men det impliserer at du nok har en gyldig tilstand fra før

Reply:
Men ja, enig i at vi kan ha `true/false` - skal bare huske å bumpe modellVersjonsnummer ettersom det ikke blir bakoverkompatibelt